src/modules/UI/components/Buttons/style.js

```diff
@@ -5,7 +5,11 @@ import { StyleSheet } from 'react-native'
 import { THEME } from '../../../../theme/variables/airbitz'
 import { scale } from '../../../../util/scaling.js'
 
-export const rawStyles = {
+export const primaryButtonUnderlay = { color: THEME.COLORS.PRIMARY }
+export const secondaryButtonUnderlay = { color: THEME.COLORS.GRAY_1 }
+export const tertiaryButtonUnderlay = { color: THEME.COLORS.GRAY_3 }
+
+export const styles = StyleSheet.create({
```

Comment:
@paullinator See how we have pulled the underlay exports out of the style block, and made them normal exports? This is why the files above work correctly.